internal/connector/internal/workers/processor_mgr.go

```diff
@@ -0,0 +1,239 @@
+package workers
+
+import (
+	"context"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/internal/api/dbapi"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/internal/services"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/db"
+	serviceError "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/workers"
+
+	"github.com/golang/glog"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+)
+
+// ProcessorManager represents a processor manager that periodically reconciles processor requests
+type ProcessorManager struct {
+	workers.BaseWorker
+	processorsService           services.ProcessorsService
+	processorDeploymentsService services.ProcessorDeploymentsService
+	connectorClusterService     services.ConnectorClusterService
+	lastVersion                 int64
+	db                          *db.ConnectionFactory
+	ctx                         context.Context
+}
+
+// NewProcessorManager creates a new processor manager
+func NewProcessorManager(
+	processorsService services.ProcessorsService,
+	processorDeploymentsService services.ProcessorDeploymentsService,
+	connectorClusterService services.ConnectorClusterService,
+	db *db.ConnectionFactory,
+	reconciler workers.Reconciler,
+) *ProcessorManager {
+	result := &ProcessorManager{
+		BaseWorker: workers.BaseWorker{
+			Id:         uuid.New().String(),
+			WorkerType: "processor",
+			Reconciler: reconciler,
+		},
+		processorsService:           processorsService,
+		processorDeploymentsService: processorDeploymentsService,
+		connectorClusterService:     connectorClusterService,
+		db:                          db,
+	}
+
+	return result
+}
+
+// Start initializes the processor manager to reconcile processor requests
+func (k *ProcessorManager) Start() {
+	k.StartWorker(k)
+}
+
+// Stop causes the process for reconciling processor requests to stop.
+func (k *ProcessorManager) Stop() {
+	k.StopWorker(k)
+}
+
+func (k *ProcessorManager) Reconcile() []error {
```

Comment:
All status reconciliations simply copy what existed for `connector`'s.

I assumed this to be a reasonable starting point but may need changing as I better understand.